ProcessMaker/processmaker
# FOUR-5822: Bad data in notifications table

## Issue & Reproduction Steps
- Edit or Insert any item of the "notifications" table, taking care of setting a simple string (for instance "mystring") for the "data" column . 
- In PM4 go to tasks and open any task.
- An error "Invalid JSON text in argument 1 to function json_extract"  will be displayed.


## Solution
The queries that are used to extract json information from the "data" column of the "notifications" table have been  modified so that the validity of the json string is checked first.

## How to Test
- Edit or Insert any item of the "notifications" table, taking care of setting a simple string (for instance "mystring") for the "data" column . 
- In PM4 go to tasks and open any task.

All tasks listings and editing a task should work without problems

## Related Tickets & Packages
[https://processmaker.atlassian.net/browse/FOUR-5822](https://processmaker.atlassian.net/browse/FOUR-5822)

## Code Review Checklist
